xtask/src/install.rs

```diff
@@ -0,0 +1,6 @@
+use xshell::cmd;
+
+pub fn install() {
+    cmd!("cargo install --path jormungandr").run().unwrap();
+    cmd!("cargo install --path jcli").run().unwrap();
+}
```

Comment:
Shouldn't this be in another pull request?

Reply:
uhm, how did it get there -_-
thanks